test/http_mock/index.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+module.exports = {
+	start() {
```

Comment:
What do we need this server for exactly? (just want to know)

Reply:
I use it for the actionHandler stub (our new feature for custom commands). client sendMessage -> chat server actionHandler->customer Server->chat server. So I'm mocking *customer Server*

Reply:
Ok can you please add that description as comment somewhere in file? Also we should stop the server after corresponding tests are executed. I don't see `stop()` getting called anywhere (unless I am missing something)